Experimental/CameraComponent/OrbitalCamera.cs

```diff
@@ -0,0 +1,296 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Composition;
+using Windows.UI.Composition.Scenes;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media.Media3D;
+
+namespace CameraComponent
+{
+    public sealed class OrbitalCamera : Camera
+    {
+        private Compositor compositor;
+        private FirstPersonCamera fp_cam;
+
+        private Vector3 target; // point in cartesian space the camera is orbiting
+        private float latitude; // vertical offset from equator of the sphere
+        private float longitude; // horizontal offset from meridian of the sphere 
+        private float radius; // distance from the point the camera is orbiting
+
+        private CompositionPropertySet propertySet;
+
+        public OrbitalCamera()
+        {
+            compositor = Window.Current.Compositor;

```

Comment:
Should the orbital camera require that the Compositor is passed in?

Reply:
I don't think this is necessary. As I understand it, there is one compositor so there is no difference between passing it to the constructor or calling Window.Current.Compositor. If my understanding is wrong or there is more than one Compositor object, please correct me.

Reply:
I'm not sure.  I know there exists a scenario where you have multiple compositors, but I don't know what that scenario is.  My other concern is that it is maybe weird to have the Camera looking upwards at the app/Window it is contained in.

Reply:
Resolved